### DISCUSSION TITLE
Got my modified SR working well. SR model imported just fine. Thank you SoftFever!

### DISCUSSION THREAD

**Discussion by MrGnarly90** [OrcaSlicer/OrcaSlicer#778] [General]
![image](https://user-images.githubusercontent.com/128745709/232575091-c4bd8c86-21f1-482e-9cb8-1a5d2e240416.png)

**Comment by Flashhhka** [comment]
Can you share your config files?

**Comment by SoftFever** [comment]
@MrGnarly90 Great! If you're interested, feel free to create a PR anytime.

**Comment by lungkao** [comment]
Can you share your config files?
